grafana/overlays/nerc-ocp-obs/grafanas/grafana.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: integreatly.org/v1alpha1
+kind: Grafana
+metadata:
+  name: grafana
+  namespace: grafana
+spec:
+  deployment:
+    envFrom:
+      - secretRef:
+          name: oauth-client-secret
+      - secretRef:
+          name: grafana-serviceaccount-token
+      - configMapRef:
+          name: openshift-service-ca.crt
+  config:
+    server:
+      root_url: https://grafana.apps.nerc-ocp-obs.rc.fas.harvard.edu
+    auth.generic_oauth:
+      enabled: true
+      scopes: openid email groups profile
+      email_attribute_path: name
+      api_url: https://dex-dex.apps.nerc-ocp-infra.rc.fas.harvard.edu/userinfo
+      auth_url: https://dex-dex.apps.nerc-ocp-infra.rc.fas.harvard.edu/auth
+      token_url: https://dex-dex.apps.nerc-ocp-infra.rc.fas.harvard.edu/token
+      role_attribute_path: >-
+        contains(groups[*], 'cluster-admins') && 'Admin' ||
+        contains(groups[*], 'nerc-org-admins')  && 'Admin' ||
+        contains(groups[*], 'nerc-logs-metrics')  && 'Admin' ||
+        contains(groups[*], 'nerc-ops')   && 'Editor' ||
```

Comment:
I would remove `nerc-org-admins` here and grant `Admin` privileges to `nerc-ops`.